norduser/grpc_interceptor.go

```diff
@@ -13,10 +13,10 @@ import (
 
 // StartNorduserdMiddleware provides a way to start/stop norduserd when handling nordvpnd gRPCs.
 type StartNorduserdMiddleware struct {
```

Comment:
if it is used in daemon, maybe we should implement thread safety measures?

Reply:
Combined norduser service is thread safe.